frontend/models/product-list/index.ts

```diff
@@ -45,7 +45,8 @@ export type {
  * Get the product list data from the API
  */
 export async function findProductList(
-   filters: ProductListFiltersInput
+   filters: ProductListFiltersInput,
+   deviceItemType: string | null = null
```

Comment:
```suggestion
   deviceItemType?: string
```

Reply:
Or maybe you wanted it to be null, I just remember you saying something like "where does this dang question mark go?"